### ISSUE TITLE
[FEATURE] Mapping Sticks to Keyboard

### ISSUE THREAD

**Issue by iamniks** [mKenfenheuer/steam-deck-windows-usermode-driver#48]
There are a bunch of games that do not have controller support, please implement mapping of joysticks to keyboards (e.g, left stick to WASD).

**Comment by trentondyck** [comment]
This would be amazing exactly what I'm looking for on the Deck playing ffxi.  I also need to map the right joystick to the arrow keys.

**Comment by lyndonguitar** [comment]
It would be amazing indeed. Binding the analog sticks to WASD or specific keyboard buttons would be nice

**Comment by Szab0t** [comment]
Yes, totally agree. This is the only missing function of the software.
